docs/EndUserPersona.md

```diff
@@ -0,0 +1,157 @@
+---
+layout: page
+title: EndUserPersona
+---
+* Table of Contents
+{:toc}
+
+# EndUserPersona
+
+This page documents the Persona of our target audiences, and how we would like to bring value to them through 
+`Inventoryinator`.
+
+Through this, we come up with a set of UserStories we would like to implement as features in `Inventoryinator` 
+
+## Persona
```

Comment:
optionally provide a short intro on how we structured our personas (explain that our app will help transition the before scenario to the after scenario, etc.)